### DISCUSSION TITLE
请问这个图片是怎么得到的

### DISCUSSION THREAD

**Discussion by ZhaoZiqi7** [KittenCN/stock_prediction#24] [General]
![image](https://github.com/KittenCN/stock_prediction/assets/127832717/93c9da76-e295-448f-ba33-037216cf0688)
这个图片是怎么得到的 不是特别会用

**Comment by KittenCN** [comment]
K线的相关代码，我暂时关掉了。。

**Comment by Zhouhuihui129** [comment]
调整 train.py 和 init.py 中的参数，先使用predict..py训练模型，生成模型文件，再使用 predict.py 进行预测，生成预测结果或测试比照图          predict..py这个是啥啊没看到有啊，求问
